libs/designer-ui/src/lib/editor/base/index.tsx

```diff
@@ -190,7 +196,12 @@ export const BaseEditor = ({
         ) : null}
         <TextPlugin
           contentEditable={
-            <ContentEditable className={css('editor-input', readonly && 'readonly')} ariaLabelledBy={labelId} ariaDescribedBy={id} />
+            <ContentEditable
+              className={css('editor-input', readonly && 'readonly')}
+              ariaLabelledBy={labelId}
+              ariaLabel={textboxBody}
```

Comment:
Remove this line. It’s redundant. LabeledBy does a better job and the message you’re using to put in here is redundant since field type is already announced